aiida_quantumespresso/utils/pseudopotential.py

```diff
@@ -1,6 +1,10 @@
 # -*- coding: utf-8 -*-
 """Utilities for pseudo potentials."""
-from aiida.orm.nodes.data.upf import UpfData, get_pseudos_from_structure
+from aiida.orm.nodes.data.upf import UpfData as LegacyUpfData
```

Comment:
In the `aiida_quantumespresso/calculations/__init__.py` file, you just load the `aiida.orm` package and use `orm.UpfData` to get what here is referred to as `LegacyUpfData`. Why not consistently use the `DataFactory` for both:

```Python
LegacyUpfData = DataFactory('upf')
UpfData = DataFactory('pseudo.upf')
```

Also in the `aiida_quantumespresso/calculations/__init__.py` file?